.github/workflows/integration-tests-vm-type.yml

```diff
@@ -100,10 +100,15 @@ jobs:
           copy_secret_to_file "$GCP_SSH_KEY" "$HOME/.ssh/GCP_SSH_KEY" 0600
           copy_secret_to_file "$GCP_SSH_KEY_PUB" "$HOME/.ssh/GCP_SSH_KEY.pub" 0600
           copy_secret_to_file "$IBM_S390X_SSH_KEY" "$HOME/.ssh/acs-sshkey_rsa.prv" 0600
+          
+          if [[ '${{ inputs.vm_type }}' =~ ppc64le ]]; then
```

Comment:
I think we can just unconditionally write the key to file (like with s390x above)

Reply:
Will it not overrite the s390x key while we will run for 390x because w are using same file name ?$HOME/.ssh/acs-sshkey_rsa.prv , and  i m guessing it will take latest one

Reply:
ah yeah I see what you mean - in that case I think this is fine. Ideally we could have different filenames for each key, but we can handle that in the future if needed.

Reply:
yeah ok